src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1108,6 +1107,10 @@ static void glb_cancel_pick_locked(grpc_exec_ctx *exec_ctx, grpc_lb_policy *pol,
     }
     pp = next;
   }
+  if (glb_policy->rr_policy != NULL) {
+    grpc_lb_policy_cancel_pick_locked(exec_ctx, glb_policy->rr_policy, target,
```

Comment:
When I first read this, I thought that this could cause us to invoke the pick callback twice, once from the pending picks list (in the loop immediately above) and again here.  It took me a little while to figure out why that wasn't the case.

It's probably worth leaving a comment somewhere in this function explaining that we will only do one of (a) invoke the callback ourselves from the pending picks list and (b) cancel the pick on the RR policy, because `create_rr_locked()` will remove entries from `glb_policy->pending_picks` before sending them to the RR policy.